sklearn/covariance/graph_lasso_.py

```diff
@@ -461,8 +461,8 @@ class GraphLassoCV(GraphLasso):
         values of alphas are passed.
 
     cv : cross-validation generator, optional
-        see sklearn.cross_validation module. If None is passed, defaults to
-        a 3-fold strategy
+        see sklearn.model_selection.split module.
```

Comment:
The documentation of cv parameters for modules outside the `model_selection` module is I feel out of scope for this PR... I'll raise a separate PR for the same after this gets merged? :) There is a PR open for that at - #4539 I'll take over that after this PR...